microcosm_flask/routing.py

```diff
@@ -77,8 +77,10 @@ def decorator(func):
 
             if enable_metrics or ns.enable_metrics:
                 from microcosm_flask.metrics import StatusCodeClassifier
-                func = graph.metrics_counting(endpoint, classifier_cls=StatusCodeClassifier)(func)
-                func = graph.metrics_timing(endpoint)(func)
+                func = graph.metrics_counting("microcosm_flask",
+                                              tags=["endpoint:" + endpoint],
+                                              classifier_cls=StatusCodeClassifier)(func)
+                func = graph.metrics_timing("microcosm_flask", tags=["endpoint:" + endpoint])(func)
```

Comment:
You should a) use f-strings instead of `+` and b) assign `tags=[f"endpoint:{endpoint}"]` so you don't need to declare this twice.